claimManagement/src/main/java/org/openimis/imisclaims/SQLHandler.java

```diff
@@ -9,17 +9,19 @@
 import android.database.sqlite.SQLiteOpenHelper;
 import android.util.Log;
 
+import static android.database.DatabaseUtils.sqlEscapeString;
+
 /**
  * Created by user on 10/01/2018.
  */
 
 public class SQLHandler extends SQLiteOpenHelper{
 
 	private static final String DB_NAME = MainActivity.Path + "Mapping.db3";
-	private static final String CreateTable = "CREATE TABLE tblMapping(Code text,Name text,Type text);";
-	private static final String CreateTableControls = "CREATE TABLE tblControls(FieldName text, Adjustibility text);";
-	private static final String CreateTableClaimAdmins = "CREATE TABLE tblAdministrators(Code text, Name text);";
-	private static final String CreateTableReferences = "CREATE TABLE tblReferences(Code text, Name text, Type text, Price text);";
+	private static final String CreateTable = "CREATE TABLE IF NOT EXISTS tblMapping(Code text,Name text,Type text);";
+	private static final String CreateTableControls = "CREATE TABLE IF NOT EXISTS tblControls(FieldName text, Adjustibility text);";
+	private static final String CreateTableClaimAdmins = "CREATE TABLE IF NOT EXISTS tblAdministrators(Code text, Name text);";
```

Comment:
I asked to change tblAdministrators to tblClaimAdmins in another PR. Please do these changes to be more precise. Thanks.

Reply:
I changed the table name to tblClaimAdmins

Reply:
I changed that.